src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxGatewayIPAddressInformation.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    /// Provides information about a network interface address.
+    internal class LinuxGatewayIPAddressInformation : GatewayIPAddressInformation
```

Comment:
Given that these are internal types, I didn't feel that this was necessary. Let me know if you feel strongly, I can make the change.